src/utils/helpers/utilityFunctions.ts

```diff
@@ -51,3 +51,15 @@ export const replaceCommas = (value: string) => value.replace(new RegExp(MAGIC_S
 
 export const removeCommasArray = (values: string[]) => values.map((value) => removeCommas(value))
 export const replaceCommasArray = (values: string[]) => values.map((value) => replaceCommas(value))
+
+// Constructs OR filter for an objects-array i.e. [ {fieldName1: value1}, {fieldName1: value2} ]
+// Returns the GraphQL filter i.e: { or: {fieldName1: { equalsTo: value1 }}, {fieldName1: { equalsTo: value2} }}}
+// This is useful to filter same key with many values using OR statement
+export const constructOrObjectFilters = (filters: { [key: string]: string }[]) => ({
```

Comment:
There's already a `constructOrFilter` function in `buildQueryFilters` that does pretty much exactly the same thing (just with slightly different input parmeters). Any chance you can just export and use that (and change what you pass to it slightly)?